src/main/java/com/mrdb/MrdbVersion1Application.java

```diff
@@ -0,0 +1,13 @@
+package com.mrdb;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class MrdbVersion1Application {
```

Comment:
Including version numbers in this main application class is a bad idea. When a new version of the app is published, the class name needs to be updated to Version2. So, we should rename this to be more generic
Just - `MrdbApplication`